fdbserver/fdbserver.actor.cpp

```diff
@@ -113,40 +113,40 @@ enum {
 	OPT_METRICSPREFIX, OPT_LOGGROUP, OPT_LOCALITY, OPT_IO_TRUST_SECONDS, OPT_IO_TRUST_WARN_ONLY, OPT_FILESYSTEM, OPT_PROFILER_RSS_SIZE, OPT_KVFILE,
 	OPT_TRACE_FORMAT, OPT_WHITELIST_BINPATH, OPT_BLOB_CREDENTIAL_FILE, OPT_CONFIG_PATH, OPT_USE_TEST_CONFIG_DB, OPT_FAULT_INJECTION, OPT_PROFILER, OPT_PRINT_SIMTIME,
 	OPT_FLOW_PROCESS_NAME, OPT_FLOW_PROCESS_ENDPOINT, OPT_IP_TRUSTED_MASK, OPT_KMS_CONN_DISCOVERY_URL_FILE, OPT_KMS_CONNECTOR_TYPE, OPT_KMS_CONN_VALIDATION_TOKEN_DETAILS,
-	OPT_KMS_CONN_GET_ENCRYPTION_KEYS_ENDPOINT, OPT_NEW_CLUSTER_KEY, OPT_USE_FUTURE_PROTOCOL_VERSION
+	OPT_KMS_CONN_GET_ENCRYPTION_KEYS_ENDPOINT, OPT_NEW_CLUSTER_KEY, OPT_AUTHZ_PUBLIC_KEY_FILE, OPT_USE_FUTURE_PROTOCOL_VERSION
 };
 
 CSimpleOpt::SOption g_rgOptions[] = {
-	{ OPT_CONNFILE,              "-C",                          SO_REQ_SEP },
-	{ OPT_CONNFILE,              "--cluster-file",              SO_REQ_SEP },
-	{ OPT_SEEDCONNFILE,          "--seed-cluster-file",         SO_REQ_SEP },
-	{ OPT_SEEDCONNSTRING,        "--seed-connection-string",    SO_REQ_SEP },
-	{ OPT_ROLE,                  "-r",                          SO_REQ_SEP },
-	{ OPT_ROLE,                  "--role",                      SO_REQ_SEP },
-	{ OPT_PUBLICADDR,            "-p",                          SO_REQ_SEP },
-	{ OPT_PUBLICADDR,            "--public-address",            SO_REQ_SEP },
-	{ OPT_LISTEN,                "-l",                          SO_REQ_SEP },
-	{ OPT_LISTEN,                "--listen-address",            SO_REQ_SEP },
+	{ OPT_CONNFILE,              "-C",								SO_REQ_SEP },
```

Comment:
The formatting of this whole section seems to be off. I think you're using tabs here